src/components/Questions/Question.js

```diff
@@ -27,12 +27,11 @@ const Question = ({
           <label htmlFor={`q${index}__name`}>{`Question #${qNumber}`}</label>
```

Comment:
something is broken in the array input fields in this file. very hard to make spaces between words in an option (sometimes i can, mostly i can't), and sometimes I can't backspace to delete some letters or words.

Reply:
On line  53, its says link inside of the icon - can you remove that. I think it shouldnt be there

Reply:
Also on line 78, the label attribute should be 

label={`q${index}__options` }

theres a lot of issues of id and label on this page on some of the inputs. the id and the label should match, and the label should only have one string inside of it, not an object. I know you didn't cause this bug, but can you make sure they all match?